changes/2025-07-24_define-branch-key-store-read-consistency-semantics/background.md

```diff
@@ -0,0 +1,78 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Define Branch Key Store read consistency semantics
+
+# Definitions
+
+## Conventions used in this document
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+# Background
+
+The [branch key store](../../framework/branch-key-store.md) persists branch key versions
+in a persistent data store such as a DynamoDb table.
+
+These distributed data stores often offer configurable read consistency,
```

Comment:
For posterity, this is sometimes referred to as "read after write". Which is the specific kind of consistency we are after here.